src/integration/tests/firstTest.test.jsx

```diff
@@ -65,6 +65,20 @@ describe("Testes", () => {
     expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).not.toBeInTheDocument();
   });
 
+  it('User should not be able to enter an empty task.', () => {
+    render(<App />);
+    const taskMsg = '';
+
+    const input = screen.getByTestId('INPUT_TASK');
+    userEvent.clear(input);
+    userEvent.type(input, taskMsg);
+    const addTask = screen.getByTestId('ADD_BUTTON');
+    userEvent.click(addTask);
+   
+    expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).not.toBeInTheDocument();
```
As the scenario says "User should not be able to enter an empty task"

(I know that there is a bug and it's possible to do it but I think we could leave test failing OR change the scenario to "User should be able to enter an empty task", it's up to you :smile: )